### PR TITLE
Correct blogpost

### PR DESCRIPTION
**Pull request by amoreauCoveo**
Taken previous comments into account
Still a few questions

### CODE REVIEW

File: _posts/2019-05-17-prometheus-memory.md [hunk 1]
```diff
@@ -11,67 +11,68 @@ author:
...
+At Coveo, we use [Prometheus 2](https://prometheus.io/) for collecting all of our monitoring metrics. Prometheus is known for being able to handle millions of time series with only a few resources. So when our pod was hiting its 30Gi memory limit, we decided to dive into it to understand how memory is allocated, and get to the root of the issue.
 
 <!-- more -->
 
-Recently we ran in an issue were our prometheus pod was killed by kubenertes because it was reaching its 30Gi memory limit. Which was surprising considering the numbers of metrics we were collecting.
-For comparaison, some benchmark available on internet give the following statistics :
+Recently, we ran into an issue were our Prometheus pod was killed by Kubenertes because it was reaching its 30Gi memory limit. This surprised us, considering the amount of metrics we were collecting.
+
+For comparison, some benchmark available on the Internet gives the following statistics:
```
Comment (amoreauCoveo) [inline _posts/2019-05-17-prometheus-memory.md]: What kind of benchmarks are those? Are they benchmarks for a typical Prometheus installation, or a maximum?

File: _posts/2019-05-17-prometheus-memory.md [hunk 2]
```diff
@@ -11,67 +11,68 @@ author:
...
+While the head block is kept in memory, blocks containing older blocks are accessed through `mmap()`. This system call acts like the swap; it will link a memory region to a file. This means we can treat all the content of the database as if they were in memory without occupying any physical RAM, but also means you need to allocate plenty of memory for OS Cache if you want to query data older than fits in the head block.
 
 ### Compactions
 
-The head block is flushed to disk periodically, while at the same time, compactions to merge a few blocks together are performed to avoid the need to scan too many blocks for queries.
+The head block is flushed to disk periodically, while at the same time, compactions to merge a few blocks together are performed to avoid needing to scan too many blocks for queries.
 
 The `wal` files are only deleted once the head chunk has been flushed to disk.
```
Comment (amoreauCoveo) [inline _posts/2019-05-17-prometheus-memory.md]: I didn't get the Investigation part. I think you might want to introduce this section so it's clearer what you are trying to accomplish.

File: _posts/2019-05-17-prometheus-memory.md (hunk 1 above)
Comment (tdegiacinto) [reply _posts/2019-05-17-prometheus-memory.md]: We can say that it's for a typical installation

Comment (amoreauCoveo) [reply _posts/2019-05-17-prometheus-memory.md]: I would rephrase the sentence like this:
"For comparison, benchmarks for a typical Prometheus installation usually look something like this:

File: _posts/2019-05-17-prometheus-memory.md (hunk 2 above)
Comment (tdegiacinto) [reply _posts/2019-05-17-prometheus-memory.md]: this is already done in the first part of this document
